LibraBFT/Impl/Consensus/RecoveryData.agda

```diff
@@ -0,0 +1,35 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
+   Copyright (c) 2021, Oracle and/or its affiliates.
+   Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
+-}
+
+open import LibraBFT.ImplShared.Consensus.Types
+open import LibraBFT.Prelude
+
+module LibraBFT.Impl.Consensus.RecoveryData where
+
+postulate
+ new
+  : Maybe Vote
+  → LedgerRecoveryData
+  → List Block
+  → RootMetadata
+  → List QuorumCert
+  → Maybe TimeoutCertificate
+  → RecoveryData
+{-
+new lastVote storageLedger blocks0 rootMetadata quorumCerts0 highestTimeoutCertificate =
```

Comment:
I assume this is just commented out because it's "TODO".  Maybe add a `TODO-1` comment if it's not being addressed imminently?

Reply:
`TODO-1` added